### PR TITLE
Adding airscape fan platform docs

### PR DESCRIPTION
**Pull request by quielb**
**Description:**
Adding documentation for the new airscape platform fan component

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26889

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/fan.airscape.markdown [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+title: "Airscape Whole House Fan"
+description: "Instructions on how to integrate Airscape Whole House fans into Home Assistant."
+logo: airscape.png
+ha_category:
+  - Fan
+ha_release: 0.100
```
Comment (klaasnicolaas) [inline source/_components/fan.airscape.markdown]:
```suggestion
ha_release: "0.100"
```

File: source/_components/fan.airscape.markdown [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+The `airscape` fan platform lets you control your network connected Airscape Whole House fan.
+
+## Configuration
+
+```yaml
+# Example configuration.yaml entry
+fan:
+  - platform: airscape
+    name: "Whole House Fan"
+    host: "192.168.10.100"
```
Comment (klaasnicolaas) [inline source/_components/fan.airscape.markdown]:
```suggestion
    host: IP_ADDRESS
```

File: source/_components/fan.airscape.markdown [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+The `airscape` fan platform lets you control your network connected Airscape Whole House fan.
+
+## Configuration
+
+```yaml
+# Example configuration.yaml entry
+fan:
+  - platform: airscape
+    name: "Whole House Fan"
```
Comment (klaasnicolaas) [inline source/_components/fan.airscape.markdown]:
```suggestion
    name: NAME_FOR_ENTITY
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by klaasnicolaas** [comment]
Where is the markdown file?

**Comment by quielb** [comment]
> Where is the markdown file?

For some reason it didn't get committed to my branch when I changed the file name.  Landing now

**Pull request closed without merging**